fe1-web/src/core/components/PoPButton.tsx

```diff
@@ -0,0 +1,51 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { StyleSheet, View, ViewStyle } from 'react-native';
+import { TouchableOpacity } from 'react-native-gesture-handler';
+
+import { Border, Colors, Spacing } from '../styles';
+
+const styles = StyleSheet.create({
+  container: {
+    width: '100%',
+    marginBottom: Spacing.x1,
+  } as ViewStyle,
+  button: {
+    padding: Spacing.x05,
+    borderColor: Colors.accent,
+    backgroundColor: Colors.accent,
+    borderWidth: Border.width,
+    borderRadius: Border.radius,
+  } as ViewStyle,
+  negative: {
+    borderColor: Colors.contrast,
+  } as ViewStyle,
+});
+
+const PoPButton = (props: IPropTypes) => {
```

Comment:
Is this button without text used anywhere (I honestly didn't check)? Otherwise we can simply keep the `PoPTextButton` and remove this one.